### PR TITLE
fix(subscription): fix sublist refrence copy issue

### PR DESCRIPTION
**Pull request by oceanlvr**
### PR Checklist

If you have any questions, you can refer to the [Contributing Guide](https://github.com/emqx/MQTTX/blob/master/.github/CONTRIBUTING.md)

#### What is the current behavior?

Please describe the current behavior and link to a relevant issue.

#### Issue Number

[874](https://github.com/emqx/MQTTX/issues/874)

#### What is the new behavior?

Please describe the new behavior or provide screenshots.

#### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

#### Specific Instructions

Are there any specific instructions or things that should be known prior to review?

### CODE REVIEW

**Comment by ysfscream** [comment]
Which issue was fixed?

File: src/components/SubscriptionsList.vue [hunk 1]
```diff
@@ -365,12 +365,11 @@ export default class SubscriptionsList extends Vue {
 
   public async resubscribe() {
     this.getCurrentConnection(this.connectionId)
-    for (let i = 0; i < this.subsList.length; i++) {
-      const sub = this.subsList[i]
+    for (let sub of this.subsList) {
```
Comment (ysfscream) [inline src/components/SubscriptionsList.vue:368]: Why change to use for of?

Comment (oceanlvr) [reply src/components/SubscriptionsList.vue:368]: Which suggested by Lint

**Comment by oceanlvr** [comment]
> Which issue was fixed?

It fixed the issue #874.

**Comment by ysfscream** [comment]
@oceanlvr OK! Thanks a lot.

**Review by ysfscream — APPROVED** [review]

**Pull request merged**